privaterelay/settings.py

```diff
@@ -250,8 +250,21 @@
 API_DOCS_ENABLED = config("API_DOCS_ENABLED", False, cast=bool) or DEBUG
 if API_DOCS_ENABLED:
     INSTALLED_APPS += [
-        "drf_yasg",
+        "drf_spectacular",
+        "drf_spectacular_sidecar",
     ]
+    CSP_IMG_SRC += ["'self'", "data:", "https://cdn.redoc.ly"]
+    CSP_SCRIPT_SRC += "blob:http://127.0.0.1:8000"
```

Comment:
this adds `['b', 'l', 'o' 'b' ... ]`. It should be:
```suggestion
    CSP_SCRIPT_SRC += ["blob:http://127.0.0.1:8000"]
```

Reply:
The `"'unsafe-inline'"` is added if you have enabled silk, which is why it might already be there for you.

Looking at `./manage.py diffsettings`, this might work for a flexible blob:

```suggestion
    CSP_SCRIPT_SRC.append(f"blob:{SITE_ORIGIN}")
    if "'unsafe-inline'" not in CSP_SCRIPT_SRC:
        CSP_SCRIPT_SRC.append("'unsafe-inline'")
```

Reply:
ooooo, it was the Silk thing. okay new change fixes it even when silk isn't installed.